File: Kode/obliger/oblig2/Actor.java
package obliger.oblig2;

import java.util.List;

public class Actor {

    String id, name;
    List<String> movies;

    public Actor(String id, String name, List<String> movies) {
        this.id = id;
        this.name = name;
        this.movies = movies;
    }

    // getters
    public String getId() {
        return id;
    }

    public String getName() {
        return name;
    }

    public List<String> getMovies() {
        return movies;
    }

    @Override
    public String toString() {
        return "("+name+")";
    }
}

File: Kode/obliger/oblig2/python/Actor.py

class Actor:

    def __init__(self, id, name, mids):
        self.id = id
        self.name = name
        self.mids = mids
        self.movies = {}

    def getMids(self):
        return self.mids

    def addMovie(self, id, movie):
        self.movies[id] = movie

    def __lt__(self, other):
        return True

    def __str__(self):
        return str(self.name)

    def __repr__(self):
        return "(A)"

